packages/docz-tools/components/DoczTOC/DoczTOC.tsx

```diff
@@ -0,0 +1,56 @@
+/** @jsx jsx */
+import { jsx } from "theme-ui";
+import { Fragment } from "react";
+import { useCurrentDoc } from "docz";
+// eslint-disable-next-line import/no-internal-modules
+import { Heading } from "docz/dist/state";
+import { Card } from "@jobber/components/Card";
+import { Content } from "@jobber/components/Content";
+import * as styles from "./styles";
+
+interface DoczTOC {
+  readonly maxDepth?: 1 | 2 | 3 | 4 | 5 | 6;
+  /**
+   * An array of items to exclude.
+   */
+  readonly exclude?: string[];
+  readonly title?: string;
+}
+
+export function DoczTOC({
+  exclude,
+  maxDepth = 6,
+  title = "Table of contents",
+}: DoczTOC) {
+  const { headings } = useCurrentDoc();
+
+  return (
+    <Card title={title}>
+      <Content>
+        <ul sx={styles.list}>
+          {headings.map((heading: Heading) => {
```

Comment:
I had changed the TOC component in docs to use a filter instead so react doesn't have to render anything for excluded items. The current fragment strategy was giving me key warnings.

```tsx
        <ul className={styles.list}>
          {headings
            .filter((heading: Heading) => {
              return !(
                (maxDepth && heading.depth > maxDepth) ||
                (exclude && exclude.includes(heading.value)) ||
                (exclude && exclude.includes(heading.slug))
              );
            })
            .map((heading: Heading) => {
              const itemClassName = classNames(
                styles.link,
                styles[`l${heading.depth}`],
              );

              return (
                <li key={heading.slug} className={itemClassName}>
                  <a href={`#${heading.slug}`} className={styles.link}>
                    {heading.value}
                  </a>
                </li>
              );
            })}
        </ul>
```